modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/inputs/DateInput.es.js

```diff
@@ -51,20 +51,30 @@ class DateInput extends React.Component {
 		const date = jsDatetoYYYYMMDD(event.target.value);
 
 		if (date !== 'Invalid Date') {
-			this.setState({value: date}, () => {
```

Comment:
Hi @Stephy6 , if we are not using this component, are these changes necessary? Is this component necessary at all?

Reply:
Hmm I thought I erased, thank u